setup.py

```diff
@@ -39,6 +50,7 @@
         "mypy_extensions>=0.4.1,<1.0.0",
         "typing_extensions>=3.7.2,<4.0.0",
         "ruamel.yaml>=0.15.87",
+        "argcomplete==1.10.0",
```

Comment:
As long as they use semantic versioning, then this can be a bit looser, like:
```suggestion
        "argcomplete>=1.10.0,<2",
```

Reply:
Thanks, updated.